readme.md

```diff
@@ -27,76 +27,140 @@ GitHub Enterprise is also supported by [authorizing your own domain in the optio
 - [**Firefox** add-on](https://addons.mozilla.org/en-US/firefox/addon/refined-github-/)
 - Opera - Use [this Opera extension](https://addons.opera.com/en/extensions/details/download-chrome-extension-9/) to install the Chrome version.
 
-
 ## Highlights
 
-- [Mark issues and pull requests as unread](https://cloud.githubusercontent.com/assets/170270/18231475/bdf83e26-72e4-11e6-958f-9ce9431d80eb.png) *(They will reappear in Notifications)*
+<table>
+    <tr>
+        <th>
+            Dashboard cleanup
+        </th>
+    </tr>
+    <tr><!-- no zebra stripes thx --></tr>
+    <tr>
+        <td>
+            <img src="media/screenshot-dashboard.png">
+        </td>
+    </tr>
+</table>
+
+<table>
+    <tr>
+        <th width="50%">
+            Mark issues and pull requests as unread<br>
+            <em>(They will reappear in Notifications)</em>
+        </th>
+        <th width="50%">
+            Preserves the original Markdown when you copy text from comments
+        </th>
+    </tr>
+    <tr><!-- no zebra stripes thx --></tr>
+    <tr>
+        <td>
+            <img width="245" src="https://user-images.githubusercontent.com/1402241/27847663-963b7d7c-6171-11e7-9470-6e86d8463771.png">
+        </td>
+        <td>
+            <img width="400" src="https://user-images.githubusercontent.com/170270/27501181-0485c5d0-586c-11e7-91ad-2d0a3537b0e2.gif">
+        </td>
+    </tr>
+</table>
+
+<table>
+    <tr>
+        <th width="50%">
+            Reaction avatars
+        </th>
+        <th width="50%">
+            Comment box
+        </th>
+    </tr>
+    <tr><!-- no zebra stripes thx --></tr>
+    <tr>
+        <td>
+            <img src="media/screenshot-reactions.png">
+        </td>
+        <td>
+            <img src="media/screenshot-comment-box.png">
+        </td>
+    </tr>
+</table>
+
+<table>
+    <tr>
+        <th colspan="2">
+            Linkifies issue/PR references in code, comments and titles
+        </th>
+    </tr>
+    <tr><!-- no zebra stripes thx --></tr>
+    <tr>
+        <td>
+            <img src="https://cloud.githubusercontent.com/assets/170270/25370217/61718820-29b3-11e7-89c5-2959eaf8cac8.png">
+        </td>
+        <td>
+            <img src="https://cloud.githubusercontent.com/assets/170270/13597190/bd487ec4-e549-11e5-9521-419fa284512c.png">
+        </td>
+    </tr>
+</table>
```

Comment:
No, I just meant normal indentation of the inner tags, but now they look ok, so not sure what happened...